Assets/MixedRealityToolkit.Tools/ExtensionServiceCreator/ExtensionServiceCreator.cs

```diff
@@ -246,39 +370,44 @@ public async void LoadStoredState()
             }
         }
 
+        /// <summary>
+        /// Validate template assets
+        /// </summary>
+        /// <remarks>
+        /// Adds to items to errors log field if not valid
+        /// </remarks>
+        /// <returns>true if errors encountered, false otherwise</returns>
```

Comment:
Looks like it returns `return errors.Count == 0`, which seems like the opposite of this statement
```suggestion
        /// <returns>true if no errors encountered, false otherwise</returns>
```